.github/workflows/examples-efr32.yaml

```diff
@@ -57,25 +60,29 @@ jobs:
                    .environment/pigweed-venv/*.log
             - name: Build example EFR32 Lock App for BRD4161A
               timeout-minutes: 10
-              run:
-                  scripts/examples/gn_efr32_example.sh examples/lock-app/efr32/
-                  out/lock_app_debug BRD4161A
+              run: |
+                  scripts/examples/gn_efr32_example.sh examples/lock-app/efr32/ out/lock_app_debug BRD4161A
+                  .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py efr32 BRD4161A lock-app \
```

Comment:
Is this running in a setup where we have to hardcode that path instead of just using `python3`?